component/src/main/java/com/synopsys/integration/alert/component/tasks/ProcessingTask.java

```diff
@@ -34,29 +36,28 @@ public abstract class ProcessingTask extends StartupScheduledTask {
     private final NotificationMappingProcessor notificationMappingProcessor;
     private final JobAccessor jobAccessor;
     private final FrequencyType frequencyType;
+    private final EventManager eventManager;
 
     private OffsetDateTime lastRunTime;
 
-    public ProcessingTask(TaskScheduler taskScheduler, TaskManager taskManager, NotificationAccessor notificationAccessor, NotificationMappingProcessor notificationMappingProcessor,
-        JobAccessor jobAccessor, FrequencyType frequencyType) {
+    protected ProcessingTask(
```

Comment:
We should break these out into their own lines

Reply:
reformatted